File: Test_Coverage/processed_dataset/tests/test_HumanEval_60.py
from functions.HumanEval_60 import sum_to_n

def test_HumanEval_60():
     assert sum_to_n(1) == 1
     assert sum_to_n(6) == 21
     assert sum_to_n(11) == 66
     assert sum_to_n(30) == 465
     assert sum_to_n(100) == 5050
 

File: Test_Coverage_Java/src/test/java/test_HumanEval_60.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class HumanEval60Test {

    @Test
    public void testSumToN() {
        assertEquals(1, HumanEval60.sumToN(1));
        assertEquals(21, HumanEval60.sumToN(6));
        assertEquals(66, HumanEval60.sumToN(11));
        assertEquals(465, HumanEval60.sumToN(30));
        assertEquals(5050, HumanEval60.sumToN(100));
    }
}